src/chart/rendering/painter/drawline.cpp

```diff
@@ -44,6 +46,23 @@ drawLine::drawLine(const Geom::Line &line,
 		canvas.setBrushColor(lineColor);
 		canvas.setLineColor(lineColor);
 
+		constexpr bool supportCurve = false;
+		if constexpr (supportCurve)
```

Comment:
What is the purpose of this?

Reply:
This is how it should be implemented, but it makes a bug a lot worst, so we should not use it till the bug is not fixed.

Reply:
could you add a comment why it is in the code?

Reply:
I have made an issue about this: https://trello.com/c/VtASDMJW/415-bug-vertical-polar-line-chart-should-have-curved-lines

Reply:
I wouldn't put a comment into the code, but I could remove the switched off code part.